### DISCUSSION TITLE
Change AWS access pattern for Bedrock LLM usage, assume role

### DISCUSSION THREAD

**Discussion by j10sanders** [langfuse/langfuse#3988] [Ideas]
### Describe the feature or potential improvement

We would prefer if you weren't asking for our access keys/secrets. Instead, we'd like to be able to create a role that can be assumed your AWS account. We would authorize the AWS account and provide you with the ARN of the AWS role to assume.

**Comment by marcklingen** [comment]
Thanks for sharing! Open for contributions on this as I agree that this is a great change

**Comment by miedc** [comment]
Hi! Any updates on this? Would be incredibly useful since we cannot use API Keys due to security policies, thanks :)
